front-end/src/store/rootReducer.js

```diff
@@ -1,6 +1,23 @@
 import { combineReducers } from "redux";
-// import reducers
+
+import user from "./User/reducer.js";
+import timeOff from "./TimeOff/reducer.js";
+import shift from "./Shift/reducer.js";
+import hourOfOperation from "./HoO/reducer.js";
```

Comment:
shouldnt that be "./hourOfOperation/reducer.js"

Reply:
yeee